src/main.py

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env python
-from models.OldMainMethod import OldMainMethod
-from models.SimpleAverage import SimpleAverage
+from models.LinearRegression import LinearRegression
+from models.SupportVectorRegression import SVR
 from models.StandaloneSimilarity import StandaloneSimilarity
+from models.NeuralNetRegressor import NeuralNetRegressor
+from models.RandomForestRegressor import RandomForestRegressor
 
 if __name__ == "__main__":
+    RF = RandomForestRegressor()
+    RF.run()
 
-    # as each Model gets it's own copy of the data due to train_test_split(), we want to
-    # free up this copy as we are going to potentially run multiple models here
-    # a reference to the class in the scope will prevent GC
+    NNR = NeuralNetRegressor()
+    NNR.run()
 
-    old_main = OldMainMethod(load_location=True)
-    old_main.run()
-    del old_main
```

Comment:
are we purposefully keeping a reference to the class and not deleting it?
every model will hold its own copy of the data - especially the self.cleaned_encoded_data insances